homeassistant/components/miflora/sensor.py

```diff
@@ -58,6 +64,9 @@
         vol.Optional(CONF_MEDIAN, default=DEFAULT_MEDIAN): cv.positive_int,
         vol.Optional(CONF_FORCE_UPDATE, default=DEFAULT_FORCE_UPDATE): cv.boolean,
         vol.Optional(CONF_ADAPTER, default=DEFAULT_ADAPTER): cv.string,
+        vol.Optional(
+            CONF_GO_UNAVAILABLE_TIMEOUT, default=DEFAULT_GO_UNAVAILABLE_TIMEOUT
```

Comment:
Can we set a sane default instead of adding an option? We try to avoid options for timeouts.

Reply:
I would prefer to not hardcode this option. If the connection quality is bad you might get only valid values during the night when no wifi device is sending, so you would set this to almost one day. If you are interested in more accurate values, for example temperature might change faster, I think a much smaller value is better if your device isn't as far away.

Reply:
Ok. Code owners should comment and review too.